docs/How/integration_checklist.md

```diff
@@ -0,0 +1,102 @@
+# Steps for a New Integration
+
+## Overview
+
+Below, you will find a checklist of the steps that should be followed by any project maintainers when performing a new
+upstream integration. This is for reference by the maintainers and by any of the community interested in the process.
+
+We will do our best to keep this up to date, so it's an excellent reference for what to expect if you're waiting on an
+integration.
+
+## 1) Announce
+
+Advertise the upcoming update, where it will start from, how long it will take, and what will happen with the existing
+release branch afterwards. This should be done in the `Release Announcements` channel in the Project Mu Team.
+
+## 2) Rebase and Test
+
+Much of this process is designed to be run on every Mu repo. When deciding which repos to update first, refer to the CI
+dependencies documented in each repo.
+
+There are currently exceptions to this rule. The `mu_basecore`, `mu_silicon_arm_tiano`, and `mu_tiano_plus` repos have a
+circular dependency that must be worked with care. Refer to the integration guide videos for clarification on how these
+three should work. They should be rebased and tested prior to any of the others, starting with `mu_basecore`.
+
+### a) Prep for the Naive Rebase
+
+* Create `rebase/*_naive` branch
+* Update the readme with new branch info and update all CI badges to look for the new branch
+  * Make sure to update source and destination commits
+
+### b) Perform the Naive Rebase
+
+NOTE: This process will not be followed the same way on "pure Mu" repos such as `mu_plus`.
+For each repo, refer to the Readme for any special maintenance instructions.
+
+* Work through the Naive Rebase
+  * Make sure to pay attention to first two commits and whether any new packages/files/submodules need to be dropped
+  * Make sure to drop any new `.pytool/Plugins` that may have been added to other repos that are only needed in
+    `mu_basecore`
+* Keep notes of any major conflicts/sightings, how the were resolved, and any follow up steps that may be needed
+* Sync the Azure pipelines and other docs, .gitignore with Basecore. Try to keep things consistent where possible
+
+### c) Clean the Rebase
+
+* Create `rebase/*_1` branch and perform initial cleaning
```

Comment:
Is there value in expanding on what you mean by "cleaning"?  Resolving conflicts, dropping commits, ... ?

Reply:
This was also not clear to me.